TESTS/network/emac/mbed_lib.json

```diff
@@ -11,7 +11,7 @@
         "UBLOX_EVK_ODIN_W2": {
             "NO_SUPPORT_FOR_MULTICAST_FILTER": true
         },
-        "MTB_MXCHIP_EMW3166": {
+        "MTB_WIO_EMW3166": {
```

Comment:
Probably typo?  It should be `WIO_EMW3166`.

Reply:
Should be fixed in last commit